### DISCUSSION TITLE
Regarding custom bootstrapped termux apk

### DISCUSSION THREAD

**Discussion by tathastu871** [termux/termux-app#3301] [Q&A] [answered]
I want to creat multiple minimal termux apks for running specific linux utilitie or shellscripts 

 eg termux app with preinstalled utilities
also i want to install multiple such termux apps on same device each with different setup

is it possible

can someone give setup for fulfilling my need

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app#forking
